server/server.go

```diff
@@ -1246,6 +1249,15 @@ func (s *Server) v1QueryGet(w http.ResponseWriter, r *http.Request) {
 	}
 	qStr := qStrs[len(qStrs)-1]
 
+	unsafeBuiltins, err := validateQuery(qStr)
```

Comment:
We should apply this to the Compile API as well as the Query API.

Reply:
Done.